services/dnsmasq/dnsmasq-MacOS.sls

```diff
@@ -37,4 +33,22 @@ dnsmasq.conf:
         - user: root
         - makedirs: True
         - template: jinja
-        - source: salt://services/dnsmasq/lessonly-resolver
\ No newline at end of file
+        - source: salt://services/dnsmasq/lessonly-resolver
+
+#
+# copy the dnsmasq service plist to LaunchDaemons
+#
+/Library/LaunchDaemons/homebrew.mxcl.dnsmasq.plist:
+    file.managed:
+        - user: root
+        - makedirs: True
+        - source: /usr/local/opt/dnsmasq/homebrew.mxcl.dnsmasq.plist
+        - name: /Library/LaunchDaemons/homebrew.mxcl.dnsmasq.plist
+
+#
+# run dnsmasq on boot
+#
+dnsmasq-launch-daemon:
+    cmd.run:
+        - name: launchctl load -w /Library/LaunchDaemons/homebrew.mxcl.dnsmasq.plist
+        - runas: root
```

Comment:
This should have a requires clause pointing at the file `/Library/LaunchDaemons/homebrew.mxcl.dnsmasq.plist:` to make sure that it exists before this is run.  It might also make sense to have watch directive so it gets run anytime that file changes

Reply:
Ah, good call!